bin/setup.bat

```diff
@@ -15,10 +15,15 @@ goto finally
 
 :setup_jruby
 REM setup_java()
+set JAVA_FROM_SYMLINK=
 if not defined JAVA_HOME IF EXIST %ProgramData%\Oracle\java\javapath\java.exe (
     for /f "tokens=2 delims=[]" %%a in ('dir %ProgramData%\Oracle\java\javapath\java.exe') do @set JAVA_HOME=%%a
+    set JAVA_FROM_SYMLINK=1
 )
 if defined JAVA_HOME set JAVA_HOME=%JAVA_HOME:\bin\java.exe=%
+if "%JAVA_FROM_SYMLINK%" == "1" (
```

Comment:
@suyograo I've added the echo to show the JAVA_HOME retrieved from the symlink.

I would have added that into the first if block, but unfortunately the cmd shell does not give me a value for `%JAVA_HOME%` inside that if block. So I've added another helper variable to do the echo afterwards.

So the output looks like this if Java is installed without the JAVA_HOME system environment set.

```
PS C:\vagrant\resources\logstash\bin> .\setup.bat
Using JAVA_HOME=C:\Program Files\Java\jre1.8.0_77 retrieved from C:\ProgramData\Oracle\java\javapath\java.exe
```

And if no Java is installed or this symlink does not exist it still shows this:

```
PS C:\vagrant\resources\logstash\bin> .\setup.bat
JAVA_HOME environment variable must be set!
```